SUMMARY.md

```diff
@@ -52,6 +52,12 @@
     * [Opłata stabilizacyjna](faqs/pl/stability-fee.md)
     * [Likwidacje](faqs/pl/liquidation.md)
     * [Zarządzanie](faqs/pl/governance.md)
+  * [中文](faqs/zh/README.md)
```

Comment:
I'm not seeing the readme.md file in this commit

Reply:
@reuptaken

Reply:
I'm not including one, since as I understand it will be generated. I've removed link.